src/sst/core/rng/marsaglia.cc

```diff
@@ -11,6 +11,7 @@
 
 #include "sst_config.h"
 
+#include "rng.h"
```

Comment:
This is already included in marsaglia.cc, did you want it in both?

Reply:
We are moving to an "include what you use" model, so starting to migrate over to including files where we use the classes to meet this future requirement. The header guards should ensure we do not duplicate etc.